sensor_msgs/QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,132 @@
+
+`sensor_msgs` uses `semver` according to the recommendation for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#versioning).
+
+### Version Stability [1.ii]
+
+`sensor_msgs` is not yet at a stable version, i.e. `>= 1.0.0`.
+
+### Public API Declaration [1.iii]
+
+All message and service definition files located in `include`, `msg` and `srv` directories are considered part of the public API.
+
+### API Stability Within a Released ROS Distribution [1.iv]/[1.vi]
+
+`sensor_msgs` will not break public API within a released ROS distribution, i.e. no major releases once the ROS distribution is released.
+
+### ABI Stability Within a Released ROS Distribution [1.v]/[1.vi]
+
+`sensor_msgs` contains C++ code and is therefore concerned with ABI stability. It will not break ABI stability within a released ROS distribution.
+
+## Change Control Process [2]
+
+`sensor_msgs` follows the recommended guidelines for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#package-requirements).
+
+### Change Requests [2.i]
+
+This package requires that all changes occur through a pull request.
+
+### Contributor Origin [2.ii]
+
+This package uses DCO as its confirmation of contributor origin policy. More information can be found in [CONTRIBUTING](../CONTRIBUTING.md).
+
+### Peer Review Policy [2.iii]
+
+Following the recommended guidelines for ROS Core packages, all pull requests must have at least 1 peer review.
+
+### Continuous Integration [2.iv]
+
+All pull requests must pass CI on all [tier 1 platforms](https://www.ros.org/reps/rep-2000.html#support-tiers)
+
+### Documentation Policy [2.v]
+
+All pull requests must resolve related documentation changes before merging.
+
+## Documentation
+
+### Feature Documentation [3.i]
+
+`sensor_msgs` has a list of provided [messages and services](README.md).
+New messages and services require their own documentation in order to be added.
+
+### Public API Documentation [3.ii]
+
+`sensor_msgs` has embedded API documentation, but it is not currently hosted.
+
+### License [3.iii]
+
+The license for `sensor_msgs` is Apache 2.0, the type is declared in the [package.xml](package.xml) manifest file, and a full copy of the license is in the repository level [LICENSE](../LICENSE) file.
+
+There is an automated test which runs a linter that ensures each file has a license statement.
+
+Most recent test results can be found [here](http://build.ros2.org/view/Epr/job/Epr__common_interfaces__ubuntu_bionic_amd64/lastBuild/testReport/sensor_msgs/copyright/)
+
+### Copyright Statements [3.iv]
+
+There are no currently copyrighted source files in this package.
+
+## Testing [4]
+
+### Feature Testing [4.i]
+
+Most of the features in sensor_msgs have corresponding tests which simulate typical usage, and they are located in the `test` directory.
+New features are required to have tests before being added.
```

Comment:
I checked that there is a test for the code in the [CMakeLists.txt](https://github.com/ros2/common_interfaces/blob/master/sensor_msgs/CMakeLists.txt#L68). But some some reason I can't see the test in the [buildfarm](http://build.ros2.org/view/Epr/job/Epr__common_interfaces__ubuntu_bionic_amd64/lastBuild/testReport/sensor_msgs/

Reply:
Weird, it's not under sensor_msgs, it's under (root). Added a link.